### DISCUSSION TITLE
Is there an easy way to connect the points in the WeylChamber plot with lines?

### DISCUSSION THREAD

**Discussion by sarafs1926** [qucontrol/weylchamber#6] [Q&A]
* weylchamber version: 0.3.2
* Python version: 3.7
* Operating System: Windows 10

### Description

Is there an easy way to connect the points in the WeylChamber plot with lines? 

### What I Did
```

cart_array = np.array(cart_coords)
n=cart_array.shape[0]
color=cm.gist_rainbow(np.linspace(0,1,n))

w = WeylChamber()
w.fig_width = 8.5
w.fig_height = 30
for x in range(0,cart_array.shape[0]):
    kwargs = {'marker': 'o'}
    w.scatter(cart_array[x][0][0],cart_array[x][0][1],cart_array[x][0][2], color=color[x], alpha=0.8, s=2, label='duration %d ns' % ts[x])
w.plot()
plt.legend(fontsize=3.5)

```

**Comment by goerz** [comment]
You can use the `render` method to draw a Weyl chamber on an existing matplotlib `Axes` object:
~~~
fig = plt.figure(figsize=(8.5, 30))
ax = fig.add_subplot(111, projection='3d')
w = WeylChamber()
w.render(ax)
~~~

After that, you can still use all the 3D plotting tools that matplotlib provides before showing the figure. See https://stackoverflow.com/questions/11541123/how-can-i-make-a-simple-3d-line-with-matplotlib/11541628 for how to draw lines.
